### ISSUE TITLE
Rule proposal: `number-to-fixed-digits`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1245]
Require pass [`digits`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed#parameters) when calling [`Number#toFixed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed), most time user just forgot `digits` not mean to `0`.

## Fail

```js
(1.23).toFixed();
```

## Pass

```js
(1.23).toFixed(0);
```

**Comment by sindresorhus** [comment]
This is now accepted.
